magento/magento2
# 25065 fixed address book tab selected

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#25065: Address Book Tab is not Active after Saving the Address at My Account Page

### Manual testing scenarios (*)

1. Do Login at Fronend
2. Go to My Account Page
3. Click on "Address Book", this tab is showing as selected.
4. Add New Address or Edit Address and save it.
5. Check the "Address Book" tab is not showing as selected.

### Questions or comments


### Contribution checklist (*)
